dotcom-rendering/src/components/FetchMoreGalleriesData.importable.tsx

```diff
@@ -0,0 +1,173 @@
+import { css } from '@emotion/react';
+import { isNonNullable, isObject } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+import { decideFormat } from '../lib/articleFormat';
+import { getDataLinkNameCard } from '../lib/getDataLinkName';
+import { addDiscussionIds } from '../lib/useCommentCount';
+import { palette } from '../palette';
+import { type DCRFrontImage } from '../types/front';
+import { type MainMedia } from '../types/mainMedia';
+import type { FETrailType, TrailType } from '../types/trails';
+import { MoreGalleries } from './MoreGalleries';
+import { Placeholder } from './Placeholder';
+
+type Props = {
+	limit: number; // Limit the number of items shown (the api often returns more)
```

Comment:
Does this need to be a prop? If this is always "5" then we can probably hardcode that in the slice below?